docs/concepts/about.md

```diff
@@ -0,0 +1,65 @@
+---
+slug: /concept/about
+sidebar_position: 2
+title: 💡 About
+---
+
+## About vet
+
+[vet](https://github.com/safedep/vet) is a free and open source tool for identifying risks in open source software supply chain. It helps security engineering teams to establish policy driven guard rails against risky OSS components.
+
+```mermaid
+flowchart TD
+    subgraph Config["🔧 Configuration"]
+        A[Package Manifests]
+        B[CEL Policies]
+    end
+
+    subgraph Analysis["🔍 Analysis Engine"]
+        D[vet]
+        E[["CEL Policy Engine"]]
+        F[Insights]
+    end
+
+    subgraph Sources["📚 Trusted Sources"]
+        G[OSV Database]
+        H[OpenSSF Scorecard]
+        I[deps.dev]
+        J[Package Registries]
+    end
+
+    subgraph Cache["📚 Cache"]
+        K[Cache]
+    end
+
+    %% Connections
+    A --> D
+    B --> E
+    D --> F
+    D --> E
+    E --> F
+    F --- K
+    F --> Sources
+
+    %% Styling
+    classDef config fill:#6a0dad,stroke:#333,stroke-width:2px,color:white
+    classDef analysis fill:#6a0dad,stroke:#333,stroke-width:2px,color:white
+    classDef sources fill:#999,stroke:#333,stroke-width:1px
+    classDef results fill:#6a0dad,stroke:#333,stroke-width:2px,color:white
+
+    class A,B config
+    class D,F analysis
+    class E analysis,thick
+    class G,H,I,J sources
+    class K results
+```
+
+Vet automates the analysis of open source dependencies by combining insights from trusted sources and allowing developers to enforce custom security policies using CEL.
+
+### 🤩 vet in Action
+
+![Vet Showcase](/img/vet/vet-demo.gif)
```

Comment:
We should use a more recent demo. 

https://github.com/safedep/vet?tab=readme-ov-file#-vet-in-action

Reply:
@abhisek done